extra/end-to-end-test.rb

```diff
@@ -0,0 +1,66 @@
+require 'rest-client'
+require 'json'
+require 'pp'
+
+# subnets and domains must exist on Infoblox
+URL = "http://localhost:8448"
```

Comment:
What's on 8448?

Reply:
Oh that's smart proxy HTTP port on my setup, I have it for a very long time now, is it different by default? Oh I see it's 8000. Changing.